.circleci/config.yml

```diff
@@ -71,17 +78,11 @@ jobs:
       - save_cache:
           key: composer-v1-{{ checksum "/tmp/composer-cache.txt" }}
           paths:
-              - ~/.cache/composer
-      # Source cache - update when branch changes
-      - save_cache:
-          key: source-v1-{{ .Branch }}
-          paths:
-            - ".git"
+            - ~/.cache/composer
 
-      # Run code reviews before installing Drupal, so that tests fail earlier.
       - run:
-          name: Run code reviews
-          command: vendor/bin/phing code-review
+          name: Install the-build in the project
```

Comment:
@agentrickard @byrond I tried to commit the newest version of the CircleCI config, but it fails at `vendor/bin/phing code-review`. The previous version (latest commit) runs `vendor/bin/the-build-installer` which adds `build.xml` to the project root. The newest version doesn't run the installer before calling `vendor/bin/phing code-review`.

Since we are moving `behat.yml` out of the skeleton, I figure we probably want to fix this in `the-build` rather than committing `build.xml` here.

Reply:
@iajon I don't quite follow the logic of this comment, other than to say that yes, the order that things run needs to be settled.

Both projects needs their own circleci config files and they need to match as closely as possible, since the-build-installer will also overwrite the circle config. But it may be that the order in which things run in drupal-skeleton is slightly different.

Only the-build needs build.xml and behat.yml.